### ISSUE TITLE
Size of static libs?

### ISSUE THREAD

**Issue by jbenet** [devsisters/goquic#11]
Why are the static libs so large?

```
4.7M    lib/darwin_amd64/libcrypto.a
16M     lib/darwin_amd64/libgoquic.a
2.0M    lib/darwin_amd64/libprotobuf.a
59M     lib/darwin_amd64/libquic.a
1.6M    lib/darwin_amd64/libssl.a
5.5M    lib/linux_amd64/libcrypto.a
7.9M    lib/linux_amd64/libgoquic.a
2.4M    lib/linux_amd64/libprotobuf.a
52M     lib/linux_amd64/libquic.a
1.9M    lib/linux_amd64/libssl.a
```

**Comment by jbenet** [comment]
this means a bin wishing to bundle this lib would gain 70-80MB in size-- this is enormous and not likely the product of raw symbols. perhaps these libs have all the debugging symbols in and could be much reduced for production?

**Comment by serialx** [comment]
Yes, it is because of the debug symbols. You can always strip them out in the build script if you want.

**Comment by l2dy** [comment]
This is making the repository large. Maybe you should try Git LFS?

**Comment by gripedthumbtacks** [comment]
Yeah, I cloned the repo and realized in total it was about 1GB?!?! ;) Git LFS or release builds for the win. Or remove the binaries from being checked in via git ignore. @serialx To fix it, add this file at the root:

.gitignore

with contents:
*.a

Or just exclude the directories you know will always have large files...
